crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/rpc.rs

```diff
@@ -46,6 +46,31 @@ impl UsedResources {
                 .or_insert(0) += count;
         }
     }
+
+    pub(crate) fn subtract_syscall_counter(
+        self: &mut UsedResources,
+        syscall_counter: &SyscallCounter,
+    ) {
+        for (syscall, count) in syscall_counter {
+            let old_syscall_count = self
+                .execution_resources
+                .syscall_counter
+                .get(syscall)
+                .unwrap_or_else(|| panic!("Missing SyscallCounter entry {syscall:?}"));
```

Comment:
I think that this should be based on a set difference and not panic here

Reply:
I need the panic here, if it panics it means we have a bug. Can change the name to indicate it in the function name if u want me to